### ISSUE TITLE
Confusing error state on review page when form is removed (526ez, 686c)

### ISSUE THREAD

**Issue by darbyemadewell** [department-of-veterans-affairs/va.gov-team#138663]
## Background/Context

When VA Form 21-526EZ is missing from the "Upload Files" section, both VA From 21-526EZ and Supporting Evidence are highlighted as errors. Is this expected?

This seems confusing to me because the error is with VA From 21-526EZ, not the Supporting Evidence, but both are indicated as invalid.

## Severity

**Approximate Section 508 Severity**: N/A

**VA Severity**: N/A

**Comment by darbyemadewell** [comment]
Partially resolved by BDD/SHA work that removed Edit button functionality from the review page. Test to confirm current functionality before starting work on this ticket.
